prefect_saturn/core.py

```diff
@@ -105,10 +107,17 @@ def _hash_flow(self, flow: Flow) -> str:
         flow with a given name, in a given Prefect Cloud project, for a given
         Prefect Cloud tenant.
         """
+        prefect_version = Version(version("prefect"))
```

Comment:
Yeah you can also just import prefect and do:

```suggestion
        prefect_version = parse(prefect.__verision__)
```

Reply:
Removed the requirement for importlib and used your suggestion in https://github.com/saturncloud/prefect-saturn/pull/47/commits/7c41201bea7643b86e51076e4bc41acc5020c6dd